libs/barista-components/sunburst/src/sunburst.util.ts

```diff
@@ -0,0 +1,466 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  DtColors,
+  DT_CHART_COLOR_PALETTE_ORDERED,
+} from '@dynatrace/barista-components/theming';
+import * as d3 from 'd3-shape';
+
+export interface DtSunburstNode {
+  label: string;
+  value?: number;
+  color?: DtColors | string;
+  children?: DtSunburstNode[];
+}
+
+export enum DtSunburstValueMode {
+  ABSOLUTE = 'absolute',
+  PERCENT = 'percent',
+}
+
+export interface DtSunburstNodeInternal {
+  origin: DtSunburstNode;
+
+  id: string;
+  label: string;
+  value: number;
+  valueRelative: number;
+  children?: DtSunburstNodeInternal[];
+
+  depth: number;
+  color: DtColors | string;
+  colorHover: DtColors | string;
+  isCurrent: boolean;
+  visible: boolean;
+  active: boolean;
+  showLabel: boolean;
+}
```

Comment:
If this is exported to the consumer, we should think about renaming this interface to not include `Internal`. 
As it is only relevant of the overlay as its implicit context, maybe `DtSunburstNodeOverlayData` would be work.

Reply:
as stated above it's renamed to DtSunburstExtendedNode. I understand that for final user TooltipData fits better but for our use it's a bit weird